collector/netframework_clrmemory.go

```diff
@@ -0,0 +1,388 @@
+			[]string{"process"},
+			nil,
+		),
+		Gen2heapsize: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "gen2heap_size_bytes"),
+			"Displays the current number of bytes in generation 2.",
+			[]string{"process"},
+			nil,
+		),
+		LargeObjectHeapsize: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "loh_size_bytes"),
+			"Displays the current size, in bytes, of the Large Object Heap.",
+			[]string{"process"},
+			nil,
+		),
+		NumberBytesinallHeaps: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "allheaps_size_bytes"),
+			"Displays the sum of the Gen 1 Heap Size, Gen 2 Heap Size, and Large Object Heap Size counters.",
+			[]string{"process"},
+			nil,
+		),
+		NumberGCHandles: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "number_gc_handles"),
+			"Displays the current number of garbage collection handles in use. Garbage collection handles are handles to resources external to the common language runtime and the managed environment.",
+			[]string{"process"},
+			nil,
+		),
+		NumberGen0Collections: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "gen0_collections_total"),
+			"Displays the number of times the generation 0 objects (that is, the youngest, most recently allocated objects) are garbage collected since the application started.",
+			[]string{"process"},
+			nil,
+		),
+		NumberGen1Collections: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "gen1_collections_total"),
+			"Displays the number of times the generation 1 objects are garbage collected since the application started.",
+			[]string{"process"},
+			nil,
+		),
+		NumberGen2Collections: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "gen2_collections_total"),
+			"Displays the number of times the generation 2 objects are garbage collected since the application started.",
+			[]string{"process"},
+			nil,
+		),
+		NumberInducedGC: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "induced_gc_total"),
+			"Displays the peak number of times garbage collection was performed because of an explicit call to GC.Collect.",
+			[]string{"process"},
+			nil,
+		),
+		NumberofPinnedObjects: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "number_pinned_objects"),
+			"Displays the number of pinned objects encountered in the last garbage collection.",
+			[]string{"process"},
+			nil,
+		),
+		NumberofSinkBlocksinuse: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "number_sink_blocksinuse"),
+			"Displays the current number of synchronization blocks in use. Synchronization blocks are per-object data structures allocated for storing synchronization information. They hold weak references to managed objects and must be scanned by the garbage collector.",
+			[]string{"process"},
+			nil,
+		),
+		NumberTotalcommittedBytes: prometheus.NewDesc(
```

Comment:
Nitpicking (and probably a fault in the generator?): Capital C in Committed (and same for reserved below)

Reply:
Ah missed that one! Thank you